Kconfig.zephyr

```diff
@@ -129,7 +129,7 @@ endif # HAS_FLASH_LOAD_OFFSET
 
 config ROM_START_OFFSET
 	hex
-	prompt "ROM start offset" if !BOOTLOADER_MCUBOOT
+	prompt "ROM start offset"
```

Comment:
drop this, this should be overridden by SoC in a defconfig file

Reply:
Hmmm... fair.